tensorboard/components/tf_ng_tensorboard/header/containers/header.component.test.ts

```diff
@@ -25,12 +25,13 @@ import {provideMockStore, MockStore} from '@ngrx/store/testing';
 import {HeaderComponent} from './header.component';
 
 import {changePlugin} from '../../core/core.actions';
-import {State, CoreState} from '../../core/core.reducers';
+import {State} from '../../core/core.reducers';
 import {
   createPluginMetadata,
   createState,
   createCoreState,
 } from '../../core/testing';
+import {PluginId} from '../../types/api';
 
 /** @typehack */ import * as _typeHackStore from '@ngrx/store';
```

Comment:
what's this?  Why not `import {Foo, Bar} from '@ngrx/store';`?

Reply:
That still makes TSC under bazel complain. :man_shrugging:

Reply:
Huh ok